plugins/inputs/ltsv_log/README.md

```diff
@@ -0,0 +1,87 @@
+# ltsv_log Service Input Plugin
+
+The ltsv_log plugin gathers metrics by reading a [LTSV (Labeled Tab-separated Values)](http://ltsv.org/) formatted log file.
+It works like the `tail` command and keep reading when more logs are added.
+And when the log file is rotated, this plugin read logs until the end from the old file and then read logs from the new file.
+
+### Configuration:
+
+```toml
+[[inputs.ltsv_log]]
+  # SampleConfig
+  ## The measurement name
+  measurement = "nginx_access"
+  ## A LTSV formatted log file path
+  ## See http://ltsv.org/ for Labeled Tab-separated Values (LTSV)
+  ## Here is an example config for nginx.
+  ##
+  ##  log_format  ltsv  'time:$time_iso8601\t'
```

Comment:
Thanks for a note. I updated the sample config with the result of `telegraf -sample-config`.